static/specialcharacters.json

```diff
@@ -0,0 +1,184 @@
+{
```

Comment:
In the README, or elsewhere can you describe where this file comes from and/or if it needs to be updated?

Reply:
Done, added to README